### DISCUSSION TITLE
Stats

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#115] [Show and tell]
![stats45](https://github.com/user-attachments/assets/449fe2fb-ccea-4578-a344-fa8b1cc97ab6)

**Comment by N3C2L** [comment]
Thanks for this truly awesome piece of software! I was never been happier with a tab manager!
I think you transformed into a grasshopper long time ago 😁 
<img width="321" alt="madprops grasshopper steampunk" src="https://github.com/user-attachments/assets/50105172-fce0-4276-abb3-cb60f0802ec2">

**Reply by madprops** [reply]
HAHA thanks I love this
